source/guides/first-steps/task-script.md

```diff
@@ -1,4 +1,121 @@
-Your First Task Script (NOT YET WRITTEN)
+Your First Task Script
 ----------------------
 
-**TODO: Write-up that guides the user through writing a basic task script (just a task with a narrate line) using VS-Code, emphasizing things like file extension and placement. Also run it in-game with `/ex run`.**
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+#### Task script basics
```

Comment:
Section titles need to be titlecase

Reply:
Also, I suggest you review your `####` vs `###` usage. I'm not sure why you're _starting_ with `####` here, then later expanding up a level to `###` (generally, if you're using `###` to be a main section, and `####` as a sub-section, you shouldn't have a sub-section without a main section).